man/systemd.resource-control.xml

```diff
@@ -1545,6 +1545,24 @@ DeviceAllow=/dev/loop-control
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ManagedOOMMemoryPressureDurationSec=</varname></term>
+
+        <listitem>
+          <para>Overrides the default memory pressure duration set by
+          <citerefentry><refentrytitle>oomd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+          this unit (cgroup). The specified value expects a time unit such as <literal>ms</literal> or
```

Comment:
expects? not really, it is fine if you don't specify a time unit, it assumes seconds in that case, that's what the `Sec=` suffix should indicate after all.

hence maybe just replace `expects` → `supports`